tests/unit/plugins/modules/test_user.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2020- IBM, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import unittest
+from unittest import mock
+import re
+import copy
+
+from ansible_collections.ibm.power_aix.plugins.modules import user
+
+from .common.utils import (
+    AnsibleFailJson, fail_json, rootdir
+)
+
+
+class Testcreate_user(unittest.TestCase):
+    def setUp(self):
+        self.module = mock.Mock()
+        self.module.fail_json = fail_json
+
+    def _setup_module_params(self):
```

Comment:
it seems like you are using _setup_module_params on every test case. i think it would be appropriate to just merge this method to the setUp method